ldb/migrations/0001_squashed_0012_auto_20170106_1510.py

```diff
@@ -393,24 +393,6 @@ class Migration(migrations.Migration):
                     "machazine",
                     models.BooleanField(default=True, verbose_name="MaCHazine"),
                 ),
-                (
-                    "board_invites",
```

Comment:
We need a new migration to remove the columns from the existing database, [`./manage.py makemigrations`](https://docs.djangoproject.com/en/4.2/ref/django-admin/#django-admin-makemigrations) should fix that